src/chord.ts

```diff
@@ -202,11 +202,11 @@ class Chord implements ChordProperties {
    * Converts the chord to a string, eg `Esus4/G#` or `1sus4/#3`
```

Comment:
```suggestion
   * Converts the chord to a string, eg `Esus4/G#` or `1sus4/#3`
   * @param {Object} [configuration={}] options
   * @param {boolean} [configuration.unicodeModifier=false] Whether or not to use unicode modifiers. 
   * This will make `#` (sharp) look like `♯` and `b` (flat) look like `♭`
```